src/main/kotlin/app/hashers/CodeLongevity.kt

```diff
@@ -73,8 +73,14 @@ class CodeLine(val repo: Repository,
     /**
      * The code line's age in seconds.
      */
-    val age : Long
-        get() = (to.commit.commitTime - from.commit.commitTime).toLong()
+    var age : Long
+        get() {
+            if (_age == 0L) {
```

Comment:
Why using backing field instead of "field" keyword?

Reply:
not sure I have a good background on this and even familiar with this, what would be your recommendation?